docs/using.rst

```diff
@@ -164,6 +172,34 @@ the Internet on the specified port using each requested domain name.
 .. note:: The ``--standalone-supported-challenges`` option has been
    deprecated since ``certbot`` version 0.9.0.
 
+.. _dns_plugins:
+
+DNS Plugins
+-----------
+
+If you'd like to obtain a wildcard certificate from Let's Encrypt or run
+``certbot`` on a machine other than your target webserver, you can use one of
+Certbot's DNS plugins.
+
+These plugins are still in the process of being packaged
+by many distributions and cannot currently be installed with ``certbot-auto``.
+If, however, you are comfortable installing the certificates yourself and/or
+setting up a more complex automated renewal process, you can run these plugins
```

Comment:
"setting up a different renewal process than using `certbot renew`" ? it's not actually so much harder to script it yourself.

Reply:
In my mind I was thinking about the renewal craziness around restarting your webserver, but I didn't add anything about that in this PR so I should probably just simplify this sentence too.